specification/monitor/resource-manager/microsoft.insights/preview/2018-06-01-preview/guestDiagnosticSettings_API.json

```diff
@@ -0,0 +1,580 @@
+                        }
+                    },
+                    "201": {
+                        "description": "A new guest diagnostic settings was successfully created.",
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsResource"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the guest diagnostic settings could not be created or updated.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },        
+            "get": {
+                "description": "Gets guest diagnostics settings.",
+                "tags": [
+                    "GuestDiagnosticsSettings"
+                 ],
+                "operationId": "guestDiagnosticsSettings_Get",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/DiagnosticSettingNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/GuestDiagnosticSettingsResource"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the diagnostic settings could not be retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "description": "Updates guest diagnostics settings.",                
+                "operationId": "guestDiagnosticsSettings_Update",                
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/DiagnosticSettingNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "diagnosticSettingsPatch",
```

Comment:
Could this please be renamed to just `"name: "parameters"`
We try to avoid API terms like "patch" in the clients :)